migrations/20190418120035-add_minmumAmount_column_to_tiers.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+import { map } from 'bluebird';
+const colName = 'minimumAmount';
+
+/**
+ * Add a `minimumAmount` column to tiers to remove ambiguity.
+ */
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    const colParams = {
+      type: Sequelize.INTEGER,
```

Comment:
`null` being a valid value for this column, is there any benefit for us to make a difference between `minAmount=0` an `minAmount=null`?

Reply:
To be clearer, is `allowNull=true` (the default) useful for us, or should we set `allowNull=false`?